.github/workflows/0-16-1-auto-merge.yaml

```diff
@@ -0,0 +1,66 @@
+name: Merge pull request into v0.16.1
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - master
+    milestone:
+      types: [created, edited]
+      names:
+        - 'v0.16.1'
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Do we need to check out the code here and below? Since all we do is invoke some GitHub APIs.